libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -580,3 +580,23 @@ AP_GPS::send_mavlink_gps2_rtk(mavlink_channel_t chan)
         drivers[1]->send_mavlink_gps_rtk(chan);
     }
 }
+
+// is this GPS instance configured
+bool AP_GPS::is_configured(uint8_t instance) const
+{
+    if (drivers[instance] != NULL && state[instance].status != NO_GPS) {
```

Comment:
This will always return false on instance 2 of single GPS operation (assuming the GPS is plugged into the normal GPS port on a Pixhawk) (I believe)